lib/pharos/command_options/load_config.rb

```diff
@@ -45,8 +45,8 @@ def default_config_yaml
             Pharos::YamlFile.new($stdin, force_erb: true, override_filename: '<stdin>')
           else
             cluster_config = Dir.glob('cluster.{yml,yml.erb}').first
-            signal_usage_error 'File does not exist: cluster.yml' if cluster_config.nil?
-            Pharos::YamlFile.new(cluster_config)
+            signal_usage_error 'File does not exist: cluster.yml' if cluster_config.nil? && !tf_json
```

Comment:
Is `tf_json` always available?

Reply:
Now it checks `.nil?`

Reply:
I meant: will `self.respond_to?(:tf_json)` always?

Reply:
Not in `terraform apply`, it only has `--config`.